### PR TITLE
Add support for dark mode/light mode toggle.

### PR DESCRIPTION
**Pull request by jckirwan**

### CODE REVIEW

File: src/slices/room.js [hunk 1]
```diff
@@ -57,6 +58,9 @@ const roomSlice = createSlice({
     toggleAudioRanges(state) {
       state.audioRangesEnabled = !state.audioRangesEnabled;
     },
+    toggleLightMode(state) {
+      state.lightMode = !state.lightMode;
+    },
```
Comment (sterling-owllabs) [inline src/slices/room.js:63]: Just writing some redux, nbd 😎 Nice! 🎉 

Under normal non-Hackathon conditions, we would likely create a new slice for settings where we'd store app-wide data/reducers like this. Def no need to change anything here though! Just noting

**Comment by sterling-owllabs** [comment]
I think the pure white grid is a little tough to look at. Can we dim that? To like 25% of what it is now? 🤔 

EDIT: Yeah, actually, we should probably let them test it out on the holo-lite monitor first. This might be just right for that.

**Comment by sterling-owllabs** [comment]
Other than that, it looks awesome!

**Review by sterling-owllabs — APPROVED** [review]

**Review by delaneypatterson — APPROVED** [review]

**Comment by jckirwan** [comment]
> I think the pure white grid is a little tough to look at. Can we dim that? To like 25% of what it is now? 🤔
> 
> EDIT: Yeah, actually, we should probably let them test it out on the holo-lite monitor first. This might be just right for that.

I actually pushed a modest change to dim it. Chances are we're going to have to make some changes once we see it on the holo-lite monitor anyway.

**Pull request merged**